extensions/couchbase/controllerservices/CouchbaseClusterService.cpp

```diff
@@ -86,19 +87,60 @@ nonstd::expected<CouchbaseUpsertResult, CouchbaseErrorType> CouchbaseClient::ups
       document_id, collection.bucket_name, collection.scope_name, collection.collection_name, upsert_err.ec(), upsert_err.message());
     return nonstd::make_unexpected(CouchbaseErrorType::FATAL);
   } else {
-    const uint64_t partition_uuid = (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->partition_uuid() : 0);
-    const uint64_t sequence_number = (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->sequence_number() : 0);
-    const uint16_t partition_id = (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->partition_id() : 0);
     return CouchbaseUpsertResult {
       collection.bucket_name,
       upsert_resp.cas().value(),
-      partition_uuid,
-      sequence_number,
-      partition_id
+      (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->sequence_number() : 0),
+      (upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->partition_uuid() : 0),
+      gsl::narrow<uint16_t>(upsert_resp.mutation_token().has_value() ? upsert_resp.mutation_token()->partition_id() : 0)
     };
   }
 }
 
+nonstd::expected<CouchbaseGetResult, CouchbaseErrorType> CouchbaseClient::get(const CouchbaseCollection& collection,
+      const std::string& document_id, CouchbaseValueType return_type) {
```

Comment:
Continuation indentation should be 4 spaces
```suggestion
nonstd::expected<CouchbaseGetResult, CouchbaseErrorType> CouchbaseClient::get(const CouchbaseCollection& collection,
    const std::string& document_id, CouchbaseValueType return_type) {
```

Reply:
Updated in https://github.com/apache/nifi-minifi-cpp/pull/1881/commits/de69b7fd973b0333b18c97b5764396ca0fe77fe0